library/common/extensions/filters/http/assertion/filter.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+#include "envoy/http/filter.h"
+
+#include "extensions/common/tap/tap_matcher.h"
+#include "extensions/filters/http/common/pass_through_filter.h"
+
+#include "library/common/extensions/filters/http/assertion/filter.pb.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Assertion {
+
+class AssertionFilterConfig {
+public:
+  AssertionFilterConfig(
+      const envoymobile::extensions::filters::http::assertion::Assertion& proto_config);
+
+  Extensions::Common::Tap::Matcher& rootMatcher() const;
+  size_t matchersSize() const { return matchers_.size(); }
+
+private:
+  std::vector<Extensions::Common::Tap::MatcherPtr> matchers_;
+};
+
+typedef std::shared_ptr<AssertionFilterConfig> AssertionFilterConfigSharedPtr;
+
+/**
+ * Harness to bridge Envoy filter invocations up to the platform layer.
+ */
```

Comment:
I don't think this is accurate since the filter isn't being bubbled up to the platform

Reply:
stale copy paste. Will refresh comment